charts/govuk-rails-app/templates/deployment.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      containers:
+        - name: {{ .Release.Name }}
+          image: "{{ .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
```

Comment:
It'd be good to mark these two values as [`required`](https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-required-function), given that the defaults are (rightly) null, so that a `helm install` without these values fails as fast as possible (render time rather than apply time).

Reply:
Good shout on this one.